src/PAModel/MsAppTest.cs

```diff
@@ -226,12 +229,14 @@ public static bool Compare(string pathToZip1, string pathToZip2, TextWriter log)
             }
 
             // If there's a checksum mismatch, do a more intensive comparison to find the difference.
-#if DEBUG
+
             // Provide a comparison that can be very specific about what the difference is.
             var comp = new Dictionary<string, byte[]>();
-            DebugChecksum(pathToZip1, log, comp, true);
-            DebugChecksum(pathToZip2, log, comp, false);
 
+            compareChecksums(pathToZip1, log, comp, true, errorContainer);
+            compareChecksums(pathToZip2, log, comp, false, errorContainer);
+
+#if DEBUG
             foreach (var kv in comp) // Remaining entries are errors.
```

Comment:
Do we still need this statement? What is this doing different that wouldn't be covered by the property comparison?